parts/kubernetesmasterresources.t

```diff
@@ -395,6 +396,7 @@
           {{end}}
         },
         "storageProfile": {
+          {{GetEtcdDisk}}
```

Comment:
@colemickens, valid point. I thought I was following/supposed to follow this pattern to get a disk but there is no need for this, I will add the JSON directly to the template file.